### PR TITLE
Add support to load the NIP05 data for a lightning address

### PR DESCRIPTION
**Pull request by bumi**
and use that nostr pubkey in the zap() function

### CODE REVIEW

**Review by rolznz — APPROVED** [review]
Nice improvement, this simplifies zapping even more 👍

**Comment by im-adithya** [comment]
> @im-adithya can you update the proxy that it also tries to load the NIP05 data? (`https://getalby.com/.well-known/nostr.json?name=adithya`)

Done: https://github.com/getAlby/lightning-address-details-proxy/pull/3

File: README.md [hunk 1]
```diff
@@ -127,11 +127,15 @@ import { LightningAddress } from "alby-tools";
 const ln = new LightningAddress("satoshi@getalby.com");
 await ln.fetch();
 
-const zapArgs = {
+if (!ln.nostrPubkey) {
+  alert('No nostr pubkey available'); // seems the lightning address is no NIP05 address
+}
+
+const
```
Comment (bumi) [inline README.md:134]:
```suggestion
const zapArgs = {
```

File: README.md [hunk 2]
```diff
@@ -127,11 +127,15 @@ import { LightningAddress } from "alby-tools";
 const ln = new LightningAddress("satoshi@getalby.com");
 await ln.fetch();
 
+if (!ln.nostrPubkey) {
+  alert('No nostr pubkey available'); // seems the lightning address is no NIP05 address
+}
+
 const zapArgs = {
+ = {
```
Comment (bumi) [inline README.md:135]:
```suggestion
const zapArgs = {
```

**Pull request merged**